components/RadioPlayer.vue

```diff
@@ -255,34 +264,20 @@ export default {
       }
     },
     show_url () {
-      const arcsiShowsList = [...this.arcsiShows]
-      let this_show
-      if (this.np.live.is_live) // live show
-      { this_show = arcsiShowsList.find(show => show.name === this.np.live.streamer_name) } else // pre-recorded show
-      { this_show = arcsiShowsList.find(show => show.name === this.np.now_playing.song.artist) }
-      let url = ''
-      try {
-        url = this_show.archive_lahmastore_base_url
-      } catch (error) { // Happens on the first call after page is loaded, not sure why...
-        // console.log("No show URL found.";
-      }
+      const url = this.currentShowArcsi ? this.currentShowArcsi.archive_lahmastore_base_url : ''
```

Comment:
Will this be a false branch if currentShowArcsi() has no hit (see above)?

Reply:
it will be empty string if this.currentShowArcsi is false/null/undefined